sui/src/validator.rs

```diff
@@ -35,8 +35,8 @@ async fn main() -> Result<(), anyhow::Error> {
         json_log_output: std::env::var("SUI_JSON_SPAN_LOGS").is_ok(),
         ..Default::default()
     };
-    #[allow(unused)]
-    let guard = telemetry_subscribers::init(config);
+
+    let _guard = telemetry_subscribers::init(config);
```

Comment:
The important thing here is that guard should not be dropped until the end of the main fn.

Reply:
@ade and if `_guard` is unused indeed, so removing the `#[allow(unused)]` will drop.

Reply:
So you want me to remove this change?

Reply:
yes please.